app/views/optOut/OptOutChooseTaxYear.scala.html

```diff
@@ -64,18 +77,19 @@
         </legend>
 
         <div class="govuk-radios" data-module="govuk-radios">
+            @availableOptOutTaxYear.zipWithIndex.map { case (optOutTaxYear, index) =>
             <div class="govuk-radios__item">
-                <input class="govuk-radios__input" id="radio1" name="optoutYear" type="radio" value="2023">
-                <label class="govuk-label govuk-radios__label" for="radio1">
-                    @messages("optout.chooseOptOutTaxYear.taxYearOnwards1")
-                </label>
-            </div>
-            <div class="govuk-radios__item">
-                <input class="govuk-radios__input" id="radio2" name="optoutYear" type="radio" value="2024">
-                <label class="govuk-label govuk-radios__label" for="radio2">
-                    @messages("optout.chooseOptOutTaxYear.taxYearOnwards2")
+                <input class="govuk-radios__input" id="radio@{index + 1}" name="optoutYear" type="radio" value="@optOutTaxYear.startYear">
```

Comment:
should fix: can we use nunjucks where ever possible

Reply:
indeed this is done in the next pr: https://github.com/hmrc/income-tax-view-change-frontend/pull/2328/files

Reply:
done